buildkite/graphql/node.graphql

```diff
@@ -0,0 +1,9 @@
+query get(
```

Comment:
Just `get` as a query name doesn't provide clarity on what it's getting.

Reply:
Yeah I like getNode

Reply:
Me thee actually, thanks for the callout - more to get around the generated Node type naming - but just a few characters longer anyway